test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -0,0 +1,89 @@
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+)
+
+const (
+	sleep = 10 * time.Minute
+)
+
+var _ = Describe("Horizontal pod autoscaling", func() {
+	var rc *ConsumingRC
+	f := NewFramework("horizontal-pod-autoscaling")
+
+	BeforeEach(func() {
+	})
+
+	AfterEach(func() {
+	})
+	It("should scale from 1 pod to 3 pods (scale resource: CPU)", func() {
+		rc = NewConsumingRC("rc", 1, "0.3", f)
+		rc.ConsumeCPU(700)
+		time.Sleep(sleep)
```

Comment:
I think this test is waiting a lot of time (it waits always 10 minutes here). I will be better to wait for no more than 10 minutes and periodically query the number of replicas. See function waitForGroupSize (resize_nodes.go:95).